modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v4_2_1/test/UpgradeCalendarBookingTest.java

```diff
@@ -55,10 +70,10 @@ public void setUp() throws Exception {
 		_upgradeDatabaseTestHelper =
 			CalendarUpgradeTestUtil.getUpgradeDatabaseTestHelper();
 		_upgradeProcess = CalendarUpgradeTestUtil.getUpgradeStep(
-			_upgradeStepRegistrator,
-			"com.liferay.calendar.internal.upgrade.v4_2_1." +
-				"CalendarBookingUpgradeProcess");
+			_upgradeStepRegistrator, _CLASS_NAME);
 		_user = UserTestUtil.addUser();
+
+		_serviceContext = createServiceContext();
```

Comment:
@rijema sort alphabetically

Reply:
_serviceContext = ...
_upgradeDatabaseTestHelper = ..
_upgradeProcess = ..
_user = ...

Reply:
I removed it . Not sure if you'll like, but I moved these instances to the createCalendar method.